lib/dataset.js

```diff
@@ -86,7 +86,7 @@ class Dataset {
             dct:title ?title .
         }
       } WHERE {
-        GRAPH <${this.graph}> {
+        GRAPH <${PUBLIC_GRAPH}> {
           ?meeting a <${VERGADERACTIVITEIT_TYPE}> ;
             besluit:geplandeStart ?start .
           BIND (CONCAT("Vergaderactiviteit in kort bestek ", ?start) AS ?title)
```

Comment:
before, we looked at a temp graph with only 1 meeting and found that 1 meeting to create a dataset.
Now we are querying all meetings in public graph, so we will create a new dataset for each meeting?

This filter should prevent that. Needs testing
```
FILTER NOT EXISTS { ?dataset dct:subject ?meeting .  } 
```

Reply:
Will this work? `FILTER NOT EXISTS { ?dataset dct:subject ?meeting }` should match a lot of datasets (all datasets that are not for this meeting), right?

Reply:
all meetings already in public graph should already have a dataset no?
This might not work for next versions of datasets

Reply:
You're right on both cases. I misread the intent of the code, we're indeed querying for a meeting here, and since we're working (in theory) with a new meeting it shouldn't have a dataset yet.

But as you say for newer datasets of an existing meeting this should result in no dataset being released.